gossip/basiccheck_test.go

```diff
@@ -10,144 +10,112 @@ import (
 	"github.com/Fantom-foundation/lachesis-base/hash"
 	"github.com/Fantom-foundation/lachesis-base/inter/idx"
 	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/stretchr/testify/suite"
+	"github.com/stretchr/testify/require"
 
 	"github.com/Fantom-foundation/go-opera/eventcheck/basiccheck"
 	"github.com/Fantom-foundation/go-opera/inter"
 )
 
-type LLRBasicCheckTestSuite struct {
-	suite.Suite
-
-	env        *testEnv
-	me         *inter.MutableEventPayload
-	startEpoch idx.Epoch
-}
-
-func (s *LLRBasicCheckTestSuite) SetupSuite() {
-	s.T().Log("setting up test suite")
+func setup(t *testing.T) (*testEnv, *inter.MutableEventPayload, idx.Epoch) {
+	t.Helper()
 
 	const (
 		validatorsNum = 10
 		startEpoch    = 1
 	)
 
-	env := newTestEnv(startEpoch, validatorsNum, s.T())
+	env := newTestEnv(startEpoch, validatorsNum, t)
 
 	em := env.emitters[0]
 	e, err := em.EmitEvent()
-	s.Require().NoError(err)
-	s.Require().NotNil(e)
+	require.NoError(t, err)
+	require.NotNil(t, e)
 
-	s.env = env
-	s.me = mutableEventPayloadFromImmutable(e)
-	s.startEpoch = idx.Epoch(startEpoch)
+	me := mutableEventPayloadFromImmutable(e)
+	return env, me, startEpoch
```

Comment:
nit: there seems to be only one call to `setup()` and it ignores the `startEpoch` result. So I guess it can be removed.